src/DataFactory/DataFactoryV2/help/Add-AzDataFactoryV2DataFlowDebugSessionPackage.md

```diff
@@ -0,0 +1,215 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.DataFactoryV2.dll-Help.xml
+Module Name: Az.DataFactory
+online version:
```

Comment:
You will need to provide `online version` for all of the new markdown files. Follow the pattern for the existing files.